### PR TITLE
feat: added credentialCard to the cell types of the table

### PR DESCRIPTION
**Pull request by sksadjad**

### CODE REVIEW

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 1]
```diff
@@ -66,6 +66,9 @@ const getCellFormatting = (type: TableCellType, value: any, opts?: TableCellOpti
     case TableCellType.STATUS: {
       return <SSIStatusLabel status={value as LabelStatus} />
     }
+    case TableCellType.REACT_ELEMENT: {
```
Comment (BtencateSphereon) [inline packages/ssi-react/src/components/views/SSITableView/index.tsx:69]: ok, i know we discussed this, but now seeing it in action i rather make this more specific. sorry about that. so i would name this CREDENTIAL_CARD or something similar. i am afraid implementers will miss use this as <SSIStatusLabel status={value as LabelStatus} /> is also a react element

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 2]
```diff
@@ -66,6 +66,9 @@ const getCellFormatting = (type: TableCellType, value: any, opts?: TableCellOpti
...
+      return value
```
Comment (BtencateSphereon) [inline packages/ssi-react/src/components/views/SSITableView/index.tsx:70]: i would then also just return the credential card here that gets an object with the required data

File: packages/ssi-react/src/components/views/SSITableView/index.tsx (hunk 1 above)
Comment (sksadjad) [reply packages/ssi-react/src/components/views/SSITableView/index.tsx]: modified to `CREDENTIAL_CARD`

File: packages/ssi-react/src/components/views/SSITableView/index.tsx (hunk 2 above)
Comment (sksadjad) [reply packages/ssi-react/src/components/views/SSITableView/index.tsx]: changed to `<SSICredentialMiniCardView {...value}/>`

**Review by BtencateSphereon — APPROVED** [review]

**Pull request merged**